### PR TITLE
[ID-59]Set appID and orgID for notifications

### LINKED ISSUE #59: Set appID and orgID for notifications
Setting up the env variables for the NotoficationsBB dependances

### PR DESCRIPTION
**Pull request by stefanvit**
## Description
Adding the enviorment variables for appID and orgID used for the NotificationBB dependances.

**Resolves #59 

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

- [x] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
Set new env variables for appID and orgID

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -76,8 +76,10 @@ func main() {
 	groupsBBAdapter := groups.NewGroupsAdapter(groupsHost, internalAPIKey)
 
 	//notifications BB adapter
+	ntAppID := getEnvKey("LMS_NOTIFICATIONS_APP_ID", true)
```
Comment (petyos) [inline main.go:79]: Let's name them LMS_ORG_ID and LMS_APP_ID as they are not directly related with the notifications BB but are related with the multi tenancy support.

**Review by petyos — CHANGES_REQUESTED** [review]
You have not done any changes of how the Notifications BB is called. You have added org id and app but do not use them anywhere.

File: driven/notifications/adapter.go [hunk 2]
```diff
@@ -45,6 +47,8 @@ func (a *Adapter) SendNotifications(recipients []core.Recipient, text string, bo
 			"subject":    text,
 			"body":       body,
 			"data":       data,
+			"appID":      a.appID,
```
Comment (petyos) [inline driven/notifications/adapter.go:50]: Notifications BB expects "org_id" and "app_id" format.

**Review by petyos — CHANGES_REQUESTED** [review]

**Review by petyos — DISMISSED** [review]
LGTM

**Review by petyos — DISMISSED** [review]
LGTM

**Review by petyos — DISMISSED** [review]
Please revert go.mod, test and then merge. go.mod should not be modified for this change.

**Review by petyos — APPROVED** [review]

**Review by mdryankov — APPROVED** [review]

**Pull request merged**